textproc/stanford-corenlp/Portfile

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           java    1.0
+
+name                stanford-corenlp
+version             3.9.2
+revision            0
+github.setup        stanfordnlp CoreNLP ${version} v
```

Comment:
Same comments as above. Move `github.setup` to before the `name` line and add the `version` there.

Reply:
Done—thanks.